applications/crossbar/src/modules/cb_signup.erl

```diff
@@ -134,16 +134,12 @@ resource_exists(_) -> 'true'.
 -spec validate(cb_context:context()) -> cb_context:context().
 -spec validate(cb_context:context(), path_token()) -> cb_context:context().
 validate(Context) ->
-    case cb_context:req_verb(Context) of
-        ?HTTP_PUT ->
-            validate_new_signup(cb_context:set_account_db(Context, ?SIGNUP_DB))
-    end.
+    (?HTTP_PUT) = cb_context:req_verb(Context),
+    validate_new_signup(cb_context:set_account_db(Context, ?SIGNUP_DB)).
 
 validate(Context, ActivationKey) ->
-    case cb_context:req_verb(Context) of
-        ?HTTP_POST ->
-            check_activation_key(ActivationKey, cb_context:set_account_db(Context, ?SIGNUP_DB))
-    end.
+    (?HTTP_POST) = cb_context:req_verb(Context),
```

Comment:
Tidier uses [erl_prettypr](https://github.com/erlang/otp/blob/maint/lib/syntax_tools/src/erl_prettypr.erl) for printing and it's this program that does this.  Note that this happens only for macros.  Do not know why there are cases where it is needed -- it's definitely not needed for `?HTTP_POST`.